djmoney/models/fields.py

```diff
@@ -298,8 +298,8 @@ def deconstruct(self):
 
         if self._has_default:
             kwargs["default"] = self.default.amount
-        if self.default_currency is not None and self.default_currency != DEFAULT_CURRENCY:
-            kwargs["default_currency"] = str(self.default_currency)
+        if self.default_currency != DEFAULT_CURRENCY:
+            kwargs["default_currency"] = str(self.default_currency) if self.default_currency is not None else None
```

Comment:
I think it would be nice to have this as separate statements, so coverage info is immediately visible

Reply:
Sure, I can fix that

Reply:
@Stranger6667 I unwrapped the oneliner into a multi-line if statement at the same place. Let me know if you thought of it differently